### PR TITLE
refine TiDB on KubeSphere

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Signed-off-by: FeynmanZhou <pengfeizhou@yunify.com>

### CODE REVIEW

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 1]
```diff
@@ -8,13 +8,15 @@ author: 'Willqy, Feynman, Sherlock'
 snapshot: 'https://ap3.qingstor.com/kubesphere-website/docs/tidb-architecture.png'
 ---
 
+![](https://ap3.qingstor.com/kubesphere-website/docs/20201028212049.png)
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:11]: Add an image name?

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 2]
```diff
@@ -8,13 +8,15 @@ author: 'Willqy, Feynman, Sherlock'
...
 
-In addition to TiDB, I am also a KubeSphere user. [KubeSphere](https://kubesphere.io/) is a distributed operating system managing cloud-native applications with [Kubernetes](https://kubernetes.io/) as its kernel, providing a plug-and-play architecture for the seamless integration of third-party applications to boost its ecosystem. [KubeSphere can be run anywhere](https://kubesphere.io/docs/introduction/what-is-kubesphere/#run-kubesphere-everywhere) as it is highly pluggable without any hacking into Kubernetes.
+In addition to TiDB, I am also a KubeSphere user. [KubeSphere](https://kubesphere.io/) is an open source distributed operating system managing cloud-native applications with [Kubernetes](https://kubernetes.io/) as its kernel, providing a plug-and-play architecture for the seamless integration of third-party applications to boost its ecosystem. [KubeSphere can be run anywhere](https://kubesphere.io/docs/introduction/what-is-kubesphere/#run-kubesphere-everywhere) as it is highly pluggable without any hacking into Kubernetes.
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:19]: open source -> open-source

Both versions are correct. Add "-" to remain consistency in this article.

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 3]
```diff
@@ -8,13 +8,15 @@ author: 'Willqy, Feynman, Sherlock'
 snapshot: 'https://ap3.qingstor.com/kubesphere-website/docs/tidb-architecture.png'
 ---
 
+![](https://ap3.qingstor.com/kubesphere-website/docs/20201028212049.png)
+
 In a world where Kubernetes has become the de facto standard to build application services that span multiple containers, running a cloud-native distributed database represents an important part of the experience of using Kubernetes. In this connection, [TiDB](https://github.com/pingcap/tidb), as an open-source NewSQL database that supports Hybrid Transactional and Analytical Processing (HTAP) workloads, has come to my awareness. It is MySQL compatible and features horizontal scalability, strong consistency, and high availability. It strives to provide users with a one-stop database solution that covers OLTP (Online Transactional Processing), OLAP (Online Analytical Processing), and HTAP services. TiDB is suitable for various use cases that require high availability and strong consistency with large-scale data.
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:13]: Please delete "as" in "as an open-source NewSQL database that supports..."

I wrote this sentence though, "as" is a bit redundant here when I read it twice.

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 4]
```diff
@@ -28,15 +30,15 @@ Therefore, I select QingCloud Kubernetes Engine (QKE) to prepare the environment
 
 1. Log in the [web console of QingCloud](https://console.qingcloud.com/). Simply select KubeSphere (QKE) from the menu and create a Kubernetes cluster with KubeSphere installed. The platform allows you to install different components of KubeSphere. Here, we need to enable [OpenPitrix](https://github.com/openpitrix/openpitrix), which powers the app management feature in KubeSphere.
 
-![qingcloud-kubernetes-engine](https://ap3.qingstor.com/kubesphere-website/docs/20201026173924.png)
+> Note: KubeSphere can be installed on any infrastructure, we just use the QingCloud platform as an example, see [KubeSphere Documentation](https://kubesphere.io/docs/) for more details.
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:33]: ->
{{< notice note >}}
KubeSphere can be installed on any infrastructure. I just use QingCloud Platform as an example. See [KubeSphere Documentation](https://kubesphere.io/docs/) for more details.
{{</ notice >}}

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 5]
```diff
@@ -56,7 +58,7 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.6/
 
 ![add-pingcap-repo](https://ap3.qingstor.com/kubesphere-website/docs/20201026193015.png)
 
-## Deploying tidb-operator
+## Deploying TiDB-operator
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:61]: I did not capitalize t in tidb as the app is named "tidb-operator" in the repo.

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 6]
```diff
@@ -94,7 +96,7 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.6/
 
 ![tidb-deployment](https://ap3.qingstor.com/kubesphere-website/docs/20201027132001.png)
 
-## Deploying tidb-cluster
+## Deploying TiDB-cluster
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:99]: same here. tidb-cluster is what displays in the repo.

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 7]
```diff
@@ -140,7 +142,7 @@ Now that we have our apps ready, we may need to focus more on observability. Kub
 
 ![tidb-cluster-deployment-ready](https://ap3.qingstor.com/kubesphere-website/docs/20201027132450.png)
 
-2. tidb, tikv and pd are all stateful applications which can be found in **StatefulSets**. Note that tikv and tidb will be created automatically and it may take a while before displaying in the list.
+2. TiDB, TiKV and pd are all stateful applications which can be found in **StatefulSets**. Note that TiKV and TiDB will be created automatically and it may take a while before displaying in the list.
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:145]: Same here. All lowercase letters
截屏2020-10-29下午12 24 24

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 8]
```diff
@@ -152,15 +154,15 @@ Now that we have our apps ready, we may need to focus more on observability. Kub
 
 ![view-tikv-loads](https://ap3.qingstor.com/kubesphere-website/docs/20201027141541.png)
 
-5. Relevant Pods are also listed. As you can see, tidb-cluster contains three pd Pods, two tidb Pods and 3 tikv Pods.
+5. Relevant Pods are also listed. As you can see, tidb-cluster contains three pd Pods, two TiDB Pods and 3 TiKV Pods.
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:157]: same here

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 9]
```diff
@@ -152,15 +154,15 @@ Now that we have our apps ready, we may need to focus more on observability. Kub
...
 
 ![tidb-pod-list](https://ap3.qingstor.com/kubesphere-website/docs/20201027134634.png)
 
 6. Go to the **Storage** section, and you can see tikv and pd are using persistent storage.
 
 ![tidb-storage-usage](https://ap3.qingstor.com/kubesphere-website/docs/20201027133725.png)
 
-7. Volume usage is also monitored. Here is an example of tikv:
+7. Volume usage is also monitored. Here is an example of TiKV:
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:165]: same

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 10]
```diff
@@ -208,7 +210,7 @@ mysql> show databases;
 mysql> 
 ```
 
-4. Besides, tidb integrates Prometheus and Grafana to monitor performance of the database cluster. As we can see above, Grafana is being exposed through `NodePort`. After you configure necessary port forwarding rules and open its port in security groups on QingCloud Platform, you can access the Grafana UI to view metrics.
+4. Besides, TiDB integrates Prometheus and Grafana to monitor performance of the database cluster. As we can see above, Grafana is being exposed through `NodePort`. After you configure necessary port forwarding rules and open its port in security groups on QingCloud Platform, you can access the Grafana UI to view metrics.
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:213]: same

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 11]
```diff
@@ -218,10 +220,12 @@ I hope you guys all have successfully deploy TiDB. Both TiDB and KubeSphere are
 
 ## References
 
-https://docs.pingcap.com/tidb-in-kubernetes/stable/get-started
+**KubeSphere GitHub**: https://github.com/kubesphere/kubesphere 
+
+**TiDB GitHub**: https://docs.pingcap.com/tidb-in-kubernetes/stable/get-started
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:225]: This link is not TiDB GitHub. It is actually TiDB Documentation

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 12]
```diff
@@ -218,10 +220,12 @@ I hope you guys all have successfully deploy TiDB. Both TiDB and KubeSphere are
...
 
-https://docs.pingcap.com/tidb-in-kubernetes/stable/tidb-operator-overview
+**TiDB-Operator Documentation**: https://docs.pingcap.com/tidb-in-kubernetes/stable/tidb-operator-overview
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:227]: perhaps no "-"
截屏2020-10-29下午12 27 41

**Review by Sherlock113 — COMMENTED** [review]
Please have a look at my comments.

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md (hunk 1 above)
Comment (FeynmanZhou) [reply content/en/blogs/TiDB-on-KubeSphere-using-qke.md:11]: OK

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md (hunk 3 above)
Comment (FeynmanZhou) [reply content/en/blogs/TiDB-on-KubeSphere-using-qke.md]: Done

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md (hunk 2 above)
Comment (FeynmanZhou) [reply content/en/blogs/TiDB-on-KubeSphere-using-qke.md]: Done

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md (hunk 8 above)
Comment (FeynmanZhou) [reply content/en/blogs/TiDB-on-KubeSphere-using-qke.md:161]: Please refer to the TiDB's technical writer:

image

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md (hunk 11 above)
Comment (FeynmanZhou) [reply content/en/blogs/TiDB-on-KubeSphere-using-qke.md]: Updated

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md (hunk 12 above)
Comment (FeynmanZhou) [reply content/en/blogs/TiDB-on-KubeSphere-using-qke.md]: Updated

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md [hunk 13]
```diff
@@ -28,15 +30,19 @@ Therefore, I select QingCloud Kubernetes Engine (QKE) to prepare the environment
 
 1. Log in the [web console of QingCloud](https://console.qingcloud.com/). Simply select KubeSphere (QKE) from the menu and create a Kubernetes cluster with KubeSphere installed. The platform allows you to install different components of KubeSphere. Here, we need to enable [OpenPitrix](https://github.com/openpitrix/openpitrix), which powers the app management feature in KubeSphere.
 
-![qingcloud-kubernetes-engine](https://ap3.qingstor.com/kubesphere-website/docs/20201026173924.png)
+{{< notice note >}}
+KubeSphere can be installed on any infrastructure, we just use the QingCloud platform as an example, see [KubeSphere Documentation](https://kubesphere.io/docs/) for more details.
```
Comment (Sherlock113) [inline content/en/blogs/TiDB-on-KubeSphere-using-qke.md:34]: This sentence has a comma splice issue. Change it to:

KubeSphere can be installed on any infrastructure. I just use QingCloud Platform as an example. See KubeSphere Documentation for more details.

**Review by Sherlock113 — COMMENTED** [review]
Please change this. Thanks.

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md (hunk 4 above)
Comment (FeynmanZhou) [reply content/en/blogs/TiDB-on-KubeSphere-using-qke.md]: Done

File: content/en/blogs/TiDB-on-KubeSphere-using-qke.md (hunk 13 above)
Comment (FeynmanZhou) [reply content/en/blogs/TiDB-on-KubeSphere-using-qke.md]: Done.

**Pull request merged**